src/corelibs/U2View/src/ov_msa/view_rendering/MaEditorSequenceArea.cpp

```diff
@@ -876,7 +876,12 @@ void MaEditorSequenceArea::mousePressEvent(QMouseEvent *e) {
             double baseWidth = ui->getBaseWidthController()->getBaseWidth();
             double baseHeight = ui->getRowHeightController()->getSingleRowHeight();
             const MaEditorSelection &selection = editor->getSelection();
-            movableBorder = SelectionModificationHelper::getMovableSide(shape, globalMousePosition, selection.toRect(), QSize(baseWidth, baseHeight));
+            movableBorder = SelectionModificationHelper::NoMovableBorder;
+            // Selection resizing is supported only in single selection mode.
+            if (selection.isSingleSelection()) {
+                const QRect &selectionRect = selection.getRectList().first();
+                movableBorder = SelectionModificationHelper::getMovableSide(shape, globalMousePosition, selectionRect, QSize(baseWidth, baseHeight));
+            }
             moveBorder(pos);
```

Comment:
Shouldn't this line be moved to the `if` statement above?

Reply:
moveBorder is safe to be called within 'non-movable' context. But you are right it can be moved inside 'if' too.  Fixed.